partner_ref_unique/README.rst

```diff
@@ -0,0 +1,59 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+========================
+Partner unique reference
+========================
+
+Add an unique constraint to partner ref field
```

Comment:
Please check it doesn't look it works

![2018-09-17_10-53-49](https://user-images.githubusercontent.com/8736623/45613929-1c5ca980-ba68-11e8-8424-cbb8bf390106.png)